src/storage/backup/src/lib.rs

```diff
@@ -73,7 +73,7 @@ impl MetaSnapshotMetadata {
             id,
             hummock_version_id: v.id,
             ssts: v.get_object_ids(),
-            max_committed_epoch: v.visible_table_committed_epoch(),
+            max_committed_epoch: v.max_committed_epoch_for_meta(),
```

Comment:
The field `max_committed_epoch` can be removed too.
Unknown fields will be ignored when deserializing MetaSnapshotMetadata.

Reply:
The field will be deprecated in https://github.com/risingwavelabs/risingwave/pull/18644